app/helpers/mb_form_builder.rb

```diff
@@ -0,0 +1,71 @@
+class MbFormBuilder < ActionView::Helpers::FormBuilder
```

Comment:
I think we need to call it something other than `FormBuilder` because that is the name of `ActionView::Helpers::FormBuilder`. Get CalFresh dealth with this by using the `GCF` prefix, which I agree is not awesome. Ideas for other names?